app.py

```diff
@@ -0,0 +1,537 @@
+
+@app.route("/contact_us")
+def contact():
+
+    return render_template("contact.html")
+
+@app.route("/register", methods=['GET', 'POST'])
+def register():
+    
+    if request.method == 'POST':
+        email_exists = mongo.db.users.find_one({'email': request.form.get('email')})
+        if email_exists is None:
+            password = request.form.get('password')
+            confirm_password = request.form.get('confirm-password')
+            if password == confirm_password:
+                hashed_pwd = bcrypt.hashpw(password=password.encode('utf-8'), salt=bcrypt.gensalt())
+                mongo.db.users.insert_one({
+                    'username': request.form.get('username'),
+                    'email': request.form.get('email'),
+                    'password': hashed_pwd.decode('utf-8')
+                })
+                flash('Account created!', category='success')
+                return redirect(url_for('login'))
+        else:
+            flash('User already exists', category="danger")        
+        return redirect(url_for('register'))       
+    return render_template("register.html")
+
+@app.route("/login", methods=['POST', 'GET'])
+def login():
+    if request.method == 'POST':
+        user_check = mongo.db.users.find_one({'email': request.form.get('email')})
+        if user_check is not None:
+            if str(user_check['email']) == str(request.form.get('email')):
+                
+                if User.validate_login(request.form.get('password'), user_check['password']):
+                    user_obj = User(user_check['email'], user_check['username'])
+                    login_user(user_obj)
+                    flash("Logged in successfully", category='success')
+                    return redirect(url_for('builds'))
+                else:
+                    flash("Incorrect E-mail/Password", category='danger')
+            else: 
+                flash("Incorrect E-mail/Password", category='danger')
+        else:
+            flash("Incorrect E-mail/Password", category='danger')
+    return render_template("login.html")
+
+@app.route("/logout")
+@login_required
+def logout():
+    logout_user()
+    flash('Logged out Successfully', category='success')
+    return redirect(url_for('login'))
+
+# CRUD Routes
+
+# Create a Record
+@app.route("/create_record")
+@login_required
+def create_record():
+    exterior = mongo.db.exterior.find()
+    engine = mongo.db.engine.find()
+    running = mongo.db.runninggear.find()
+    interior = mongo.db.interior.find()
+
+    return render_template("create.html", exterior=exterior, engine=engine, running=running, interior=interior)
+
+# Insert Record into DB
+@app.route("/insert_record", methods=['POST'])
+def insert_record():
+    builds = mongo.db.builds
+
+    exterior = mongo.db.exterior.find()
+    engine = mongo.db.engine.find()
+    running = mongo.db.runninggear.find()
+    interior = mongo.db.interior.find()
```

Comment:
ideally, you shouldn't need to make all these database queries in this route again since you made them in the  previous `/create_record` route. Grab what you need from there.